### ISSUE TITLE
Export Family Tree Chart/Graph as Image

### ISSUE THREAD

**Issue by Wdavery** [gramps-project/gramps-web#346]
Within the Family Tree view, it would be handy to be able to export as an image. 

Obviously the workaround is simply to screenshot the view, but in a large graph the resolution would end up quite low. Ideally you could export the entire graph as a PNG (with resolution dependent on the size of the graph).

**Comment by DavidMStraub** [comment]
Indeed; it is already SVG but what will be tricky is including CSS and images.

@Wdavery just to check, you're aware of the graph reports, which can achieve similar things, albeit with slightly different style?

**Comment by Wdavery** [comment]
I am yes, they are certainly a functional replacement but are nowhere near as slick as the web view.

**Comment by geostag** [comment]
I will investigate on how to generate pdf out of the svg.

**Comment by geostag** [comment]
I tried [svg2pdf](https://github.com/yWorks/svg2pdf.js) but did not manage to get it working with gramps-web. Second try is using [d3-save-svg](https://github.com/edeno/d3-save-svg). Using this, a svg file can be ![grafik](https://github.com/user-attachments/assets/7b79156b-a87a-4efe-acf4-476659b93f42) from any gramps-web chart. However there are some caveats: font is not embedded, images as well. I created a [branch](https://github.com/geostag/gramps-web/tree/exportsvg) with relationship chart directly exporting svg. Not shure how to proceed.
